edit/header-resizer.js

```diff
@@ -0,0 +1,50 @@
+/* global $ $$ */// dom.js
+/* global editor */
+/* global prefs */
+'use strict';
+
+(function HeaderResizer() {
+  const PREF_ID = 'editor.headerWidth';
+  const el = $('#header-resizer');
+  let lastW, lastX;
+  prefs.subscribe(PREF_ID, (key, val) => {
```

Comment:
updateHeaderWidth is in `base` because it must run early before the page is shown so I can't move it into header-resizer which runs after everything was shown.

I also don't want to embed more code from header-resizer into base because "base" means the basic necessities e.g. there's no need to process codemirror widgets in base because there's no codemirror yet.